CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 1.0.0 (March 8, 2017)
```

Comment:
The main argument against 1.0.0 as per semantic versioning is that some of our previous releases haven't followed that standard. 0.1.x to 0.2.0 had breaking changes, for instance.